services/wallet/router/router.go

```diff
@@ -280,7 +280,8 @@ func NewRouter(rpcClient *rpc.Client, transactor *transactions.Transactor, token
 	transfer := pathprocessor.NewTransferProcessor(rpcClient, transactor)
 	erc721Transfer := pathprocessor.NewERC721Processor(rpcClient, transactor)
 	erc1155Transfer := pathprocessor.NewERC1155Processor(rpcClient, transactor)
-	cbridge := pathprocessor.NewCelerBridgeProcessor(rpcClient, transactor, tokenManager)
+	// TODO: Celar Bridge is out of scope for 2.30, check it thoroughly once we decide to include it again
+	// cbridge := pathprocessor.NewCelerBridgeProcessor(rpcClient, transactor, tokenManager)
```

Comment:
@saledjenic This is a good example of where a feature flag would be useful and possibly better than commented out code. One reason because the code would need to compile, so there would be a higher chance that when the dev wants to re-enable support for this bridge it will work.